packages/yourloops/test/integration/use-cases/patient-data-visualisation.ts

```diff
@@ -39,12 +39,18 @@ import {
 import { patient1, patient2 } from '../data/patient.api.data'
 import { checkPatientDropdown, checkPatientNavBarForPatient } from '../assert/patient-nav-bar.assert'
 import { checkMonitoringAlertsCard } from '../assert/monitoring-alerts.assert'
+import {
+  checkEmptyMedicalFilesWidgetForHcp,
+  checkEmptyMedicalFilesWidgetForPatient
+} from '../assert/medical-widget.assert'
 
 export const testDashboardDataVisualisation = async (patientDashboardLayoutParams: PatientDashboardLayoutParams) => {
   await checkPatientDashboardLayout(patientDashboardLayoutParams)
   await checkPatientStatistics()
   await checkDeviceUsageWidget()
-  await checkMonitoringAlertsCard()
+  if (patientDashboardLayoutParams.isMonitoringAlertCardVisible) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏I personnally would prefer for the `test` function to have the less possible logic in them. IMO having a function `testDashboardDataVisualisationForPatient` and another one `testDashboardDataVisualisationForHcp` makes everything more readable and obvious